src/components/TruthTree.tsx

```diff
@@ -22,9 +22,7 @@ const TruthTree = ({ currentState, dispatch }: Props) => {
   return (
     <CustomTooltip title={!feedback.success ? feedback.errorMessage : ''}>
       <div
-        className={`${classes.NodeViewBaseContainer} ${
-          feedback.success ? '' : classes.Incorrect
```

Comment:
I prefer doing it this way, using a css class to mark something as correct or incorrect, instead of setting the borderColor inline. That way, we can adjust how we display an incorrect node by changing the stylesheet, instead of looking for where we've put styles in the components.

Also it's intentional that the border only gets set for the case where sequent-conversion fails, i.e. `feedback.success === false`. If feedback.success is true, it just means that we were able to check it. And in that case, the nodes will be individually highlighted, so no need to highlight the whole tree.